asst1/asst1.py

```diff
@@ -0,0 +1,81 @@
+# coding: utf-8
+import collections
+import sys
+
+# Return that string with the most common letter swapped with the least common letter
+def swapchars(input):
+	without_punc = ""
+	length = len(input)
+	for i in range(length):
+		if input[i].isalpha():
+			without_punc += input[i]
+	
+	char_list = collections.Counter(without_punc).most_common(26)
+	input_length = len(char_list)
+	most_common = char_list[0][0]
+	least_common = char_list[input_length - 1][0]
+
+	switched = ""
+
+	for i in range(length):
+		if input[i] == most_common.upper() or input[i] == most_common.lower():
+			switched = switched + least_common
+		elif input[i] == least_common.upper() or input[i] == least_common.lower():
+			switched = switched + most_common
+		else:
+			switched = switched + input[i]
+
+	return switched
+
+# Takes one (1) integer argument n and an arbitrary number of string arguments. 
```

Comment:
Nice commenting! Happy that you're documenting your code. We'll learn more about python documentation convention in the comp but good to see that you're commenting (: